app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss

```diff
@@ -1,25 +1,268 @@
-@import "govuk/components/accordion/accordion";
+@import "govuk_publishing_components/component_support";
 
-.gem-c-accordion {
-  .govuk-accordion__section-heading {
-    // this is a temporary addition to fix the line height when it
-    // is being overridden by styles from govuk-template in collections
-    @include govuk-media-query($until: desktop) {
-      @include govuk-font(24, $weight: bold, $line-height: 1.6);
+$gem-c-accordion-border-width: 3px;
+$gem-c-accordion-bottom-border-width: 1px;
+
+// Buttons within the sections don’t need default styling
+.gem-c-accordion__section-button {
+  display: inline-block;
+  margin-bottom: 0;
+  padding-top: govuk-spacing(3);
+  font-weight: bold;
+  @include govuk-font($size: 24, $weight: bold);
+}
+
+.gem-c-accordion__section-header {
+  padding-top: govuk-spacing(2) 0;
+}
+
+.gem-c-accordion__section-heading,
+.gem-c-accordion__section-summary {
+  margin: govuk-spacing(1) 0;
+}
+
+.js-enabled {
+  .gem-c-accordion {
+    border-bottom: $gem-c-accordion-bottom-border-width solid $govuk-border-colour;
+  }
+
+  .gem-c-accordion__controls {
+    text-align: left;
+  }
+
+  .gem-c-accordion__open-all {
+    position: relative;
+    z-index: 1;
+    border-width: 0;
+    color: $govuk-link-colour;
+    background: none;
+    -webkit-appearance: none;
+    cursor: pointer;
+    margin: 0 0 govuk-spacing(5) 0;
+    // Slight right padding needed as Chevron clipped on iOS
+    padding: 0 govuk-spacing(1) govuk-spacing(1) 0;
+    @include govuk-font($size: 16);
+    @include govuk-link-common;
+    @include govuk-link-style-default;
+    // Remove default button focus outline in Firefox
+    &::-moz-focus-inner {
+      padding: 0;
+      border: 0;
+    }
+  }
+
+  .gem-c-accordion__open-all:hover .gem-c-accordion__open-all-text {
+    color: $govuk-link-colour;
+    text-decoration: underline;
+  }
+
+  // Focus state to change chervon icon to black
+  .gem-c-accordion__open-all:focus {
+    .gem-c-accordion__open-all-text {
+      color: $govuk-focus-text-colour;
+      text-decoration: none;
+    }
+
+    .gem-c-accordion-nav__chevron--stroke {
+      stroke: govuk-colour("black");
     }
   }
-}
 
-.govuk-accordion--condensed {
-  .govuk-accordion__section-button {
-    @include govuk-media-query($from: tablet) {
-      @include govuk-font($size: 19, $weight: bold);
+  .gem-c-accordion__section-heading {
+    // Override browser defaults to ensure consistent element height
+    margin-top: 0; // Override browser default
+    margin-bottom: 0; // Override browser default
+    @include govuk-font(24);
+  }
+
+  // Section headers have a pointer cursor as an additional affordance
+  .gem-c-accordion__section-header {
+    padding: govuk-spacing(2) 0 govuk-spacing(5);
+    position: relative;
+    border-top: $gem-c-accordion-bottom-border-width solid $govuk-border-colour;
+    color: $govuk-link-colour;
+  }
+
+  .gem-c-accordion__section-header--focused {
```

Comment:
It just occurred to me that you're applying this as a class via js. Can this not be simply applied to a `:focus` state? I'm aware the focus won't be necessary if the js isn't on but if the js ever breaks post markup generation then the focus state will also be broken. I remember I had problems with this but was able to cobble it together with some creative spacing changes, check out my step by step PR or I can talk you through it.

Reply:
So, this is due to the focus state visuals changing to the parent element.  The button is being focused semantically but visually the outer element is receiving the styles.  This think this could be refactored however I'm a little hesitate as the Design could be iterated on.

Reply:
I hear you on the design iteration point, we've both chosen implementations that aren't perfect but do the job. I would argue that mine has less user risk even if it's a little messier and trickier to unpick upon change. I wonder if there's value in getting another dev to step in and take a look at our choices and see what they reckon.

Reply:
>we've both chosen implementations that aren't perfect but do the job.

This, I'm mindful that if the end goal is for this visual change to inform the singular point of truth: the Design System _and_ once there is more critical user & a11y clinic feedback there could be more significant iterations.  

In this context I'm thinking it might be more desirable to sort of perfect a _little_ later when we know "yes this is as good as it can be" type thing..

> I wonder if there's value in getting another dev to step in and take a look at our choices and see what they reckon.

I think so, that way if there are any glaring issues I / we can sort.

Reply:
With that in mind let's leave it for now. It's a low risk and a more refined solution is probably useful. Feel free to resolve.